blark/iec.lark

```diff
@@ -529,6 +538,7 @@ function_declaration: "FUNCTION"i [ access_specifier ] derived_function_name [ "
                    | output_declarations
                    | input_output_declarations
                    | static_var_declarations
+                   | external_var_declarations
```

Comment:
Checking on the transformer - no additional changes were needed because we rely on inheritance for `VariableDeclarationBlock` 👍